test/e2e/storage/vsphere_stress.go

```diff
@@ -0,0 +1,126 @@
+package storage
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	storageV1 "k8s.io/api/storage/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"os"
+	"strconv"
+	"sync"
+)
+
+/*
+	Induce stress to create volumes in parallel with multiple threads based on user configurable values for number of volumes and threads.
+	The following actions will be performed as part of this test.
+
+	1. Create Storage Classes of 4 Categories (Default, SC with Non Default Datastore, SC with SPBM Policy, SC with VSAN Storage Capalibilies.)
+    2. READ VCP_STRESS_INSTANCES and VCP_STRESS_VOLUMES_PER_INSTANCE from System Environment.
+	3. Launch goroutine for creating volumes n times. Here n is the value specified by the user in the system env VCP_STRESS_INSTANCES env variable.
+	4. Each instance of routine creates m number of volumes using the storage classes created in step-1. Here m is value specified by the user in the system env VCP_STRESS_VOLUMES_PER_INSTANCE.
+*/
+var _ = SIGDescribe("vcp-stress", func() {
+	f := framework.NewDefaultFramework("vcp-stress")
+	var (
+		client    clientset.Interface
+		namespace string
+	)
+
+	BeforeEach(func() {
+		framework.SkipUnlessProviderIs("vsphere")
+		client = f.ClientSet
+		namespace = f.Namespace.Name
+		Expect(os.Getenv("VCP_STRESS_INSTANCES")).NotTo(BeEmpty(), "ENV VCP_STRESS_INSTANCES is not set")
+		Expect(os.Getenv("VCP_STRESS_VOLUMES_PER_INSTANCE")).NotTo(BeEmpty(), "ENV VCP_STRESS_VOLUMES_PER_INSTANCE is not set")
+		Expect(os.Getenv("VSPHERE_SPBM_GOLD_POLICY")).NotTo(BeEmpty(), "ENV VSPHERE_SPBM_GOLD_POLICY is not set")
```

Comment:
Any significance of keeping env var name to "VSPHERE_SPBM_GOLD_POLICY"?
I mean why "GOLD" word is added?

Reply:
Fixed: 
```
Expect(os.Getenv("VSPHERE_SPBM_POLICY_NAME")).NotTo(BeEmpty(), "ENV VSPHERE_SPBM_POLICY_NAME is not set")
```